test/source/browser/browser-handle.ts

```diff
@@ -73,13 +73,19 @@ export class BrowserHandle {
   public newPageTriggeredBy = async (t: AvaContext, triggeringAction: () => Promise<void>): Promise<ControllablePage> => {
     const page = (await this.doAwaitTriggeredPage(triggeringAction))!;
     const url = page.url();
-    const parsedUrl = new URL(url);
+    let pathname = url;
+    try {
+      const parsedUrl = new URL(url);
+      pathname = parsedUrl.pathname;
+    } catch (e) {
+      console.error('Error parsing URL:', e.message);
```

Comment:
live tests pass now, but this error is thrown by each of them.
do you know which URL cause it?

<img width="300" alt="Screenshot 2024-08-08 at 12 05 38" src="https://github.com/user-attachments/assets/c579b026-14a2-4159-824d-6d1e270d25ba">

Reply:
Invalid url which caused above issue is `:`

Reply:
Got it, but logs look strange - tests passed but there are parsing errors, let's update handling to something like this:
```ts
let pathname: string;
try {
  pathname = new URL(url).pathname;
} catch {
  pathname = url;
}
```

Reply:
Sounds good!